yt_dlp/extractor/espn.py

```diff
@@ -353,6 +353,14 @@ def _real_extract(self, url):
             if not cookie:
                 self.raise_login_required(method='cookies')
 
+            jwt = re.search(r'=(.*)\|', cookie.value).group(1)
```

Comment:
use `_search_regex` here

```suggestion
            jwt = self._search_regex(r'=([^|]+)\|', cookie.value, 'cookie jwt')
```